src/mongo/db/query/explain.cpp

```diff
@@ -855,8 +913,9 @@ void Explain::explainStages(PlanExecutor* exec,
                             const Collection* collection,
                             ExplainOptions::Verbosity verbosity,
                             BSONObjBuilder* out) {
+    
     auto winningPlanTrialStats = Explain::getWinningPlanTrialStats(exec);
-
+   
```

Comment:
Looks like a stray whitespace change (and also on line 916), can you please undo this?